docs/SNIPPET.txt

```diff
@@ -0,0 +1,5 @@
+def hello(name=''):
+    if name:
+        return u'Hello, {}!'.format(name)
```

Comment:
I don't like this `u''` part since Strings in Python 3 are Unicode ones - https://docs.python.org/3/howto/unicode.html#python-s-unicode-support
And we should highlight the current version of the language.

Plus this code doesn't reflect OOP portion of Python

Reply:
These are all good observations. @m-a-ge would you be willing to find a better example for this?

Reply:
@kytrinyx,
I was thinking about something like this:
```python
class Greeter(object):
    name = ''
    def __init__(this, name):
        this.name = name
        
    def greet(this):
        print('Hello, {}!'.format(this.name))
        
name = input('Enter your name: ')
Greeter(name).greet()
```
@behrtam, what do you think?

Side note: I understand that extending `object` is not necessary in Python 3, but I intentionally put it there for demonstration purposes